frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -612,16 +654,19 @@ export const UpdateInProgress: React.FC<UpdateInProgressProps> = ({
       ? masterMachinePoolConfig?.status?.updatedMachineCount
       : 0;
   const percentMasterNodes = calculatePercentage(updatedMasterNodes, totalMasterNodes);
+  const { t } = useTranslation();
 
   return (
     <UpdatesProgress>
       <UpdatesGroup>
         <UpdatesType>
-          <Link to="/settings/cluster/clusteroperators">Cluster Operators</Link>
+          <Link to="/settings/cluster/clusteroperators">
+            {t('cluster-settings~Cluster operators')}
+          </Link>
         </UpdatesType>
         <UpdatesBar>
           <Progress
-            title={`${updatedOperatorsCount} of ${totalOperatorsCount}`}
+            title={`${updatedOperatorsCount} ${t('cluster-settings~of')} ${totalOperatorsCount}`}
```

Comment:
`updatedOperatorsCount` and `totalOperatorsCount` should be parameters to this message